src/request_handler.cc

```diff
@@ -1,59 +1,96 @@
+#include <string>
+#include <sstream>
 #include <boost/filesystem.hpp>
 #include <boost/filesystem/fstream.hpp>
-#include <string>
-#include <unordered_map>
 #include "request_handler.h"
 
-std::string get_mime_type(std::string extension);
+void RequestHandler::ParseConfig(const NginxConfig& config) {
+  for (auto statement : config.statements_) {
+    config_map_[statement->tokens_[0]] = statement->tokens_[1];
+  }
+}
+
+RequestHandler::Status EchoHandler::Init(const std::string& uri_prefix,
```

Comment:
uri_prefix is the value in the config file that this handler will run for.
i.e. "/echo"